release/models/wifi/openconfig-wifi-mac.yang

```diff
@@ -117,7 +123,7 @@ module openconfig-wifi-mac {
       default "oc-wifi-types:FREQ_2_5_GHZ";
```

Comment:
Does it make sense to change default to tri-band(FREQ_2_5_6_GHZ) now given most APs are now going to be 6E.